### DISCUSSION TITLE
CPS测试疑问，请求解惑

### DISCUSSION THREAD

**Discussion by BOY332** [baidu/dperf#462] [Q&A]
server配置如下：
![image](https://github.com/baidu/dperf/assets/14010909/843a9232-3c22-4aa4-bcb6-a60a400ccc79)

client配置如下：
![image](https://github.com/baidu/dperf/assets/14010909/9773fb41-e08c-4293-8620-3ac35016cbdf)

测试结果如下：
![image](https://github.com/baidu/dperf/assets/14010909/5c2738a3-4b13-4121-9646-0fdbb9e96cb7)

我的新建CPS速率是40000，看起来TX的速率是12wPPS，RX速率是12wPPS，所以一个新建连接会处理6个包对吗？但是文档上我看写的是七个包，server发三个，client发四个？看起来和数据有些对不上呢？麻烦有空解答一下我的疑惑，谢谢~
![image](https://github.com/baidu/dperf/assets/14010909/58c9f3a6-a48d-4164-973b-68929665433d)

**Comment by BOY332** [comment]
大佬，可以麻烦简单的介绍一下CPS测试发包的逻辑吗？

**Comment by pengjianzhang** [comment]
文档有错误，已经更新文档
另外你可以抓包看下
https://dperf.org/doc/html/dperf-dpdk-pdump
